### DISCUSSION TITLE
Ajout codeInsee dans l'objet Adress lié à l'équipement

### DISCUSSION THREAD

**Discussion by ericjacq92** [before-interop/malfacon#8] [General]
Peut on ajouter le codeInsee dans l'objet Adress précisant la localisation de l'équipement.
Cet élément permet d'obtenir la codification exacte de la commune, bien plus précis qu'un seimple champ city.

**Comment by ggrebert** [comment]
exemple sur le protocole AnomalieAdresse:

![image](https://github.com/before-interop/malfacon/assets/1737774/ce3abfa6-5889-428e-a921-21b2d95c5755)

**Reply by ericjacq92** [reply]
Oui il serait donc pertinent de reconduire cette structure au niveau de Malfaçon (de type GeographicAddress décrite dans les objets common)
